IntelPython/sdc
# Fix bm parallel

This PR do:
* `TestBase` inherits from `TestCase`
* `TestBase` uses `self.jit()` from `TestCase` instead of `numba.njit()`. It configures `parallel=True`.
* Remove `_test_jitted()` and `_test_python()` from perf test classes
* Use `_test_jit()` and `_test_py()` in perf test for `read_csv()`
* tests for `read_csv` should start with other tests due to importing like others.
* Sort imports of perf tests in abc order.
